src/Login.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import { getUserData } from './apiCalls'
+import { Redirect } from 'react-router-dom';
+import './Login.css';
+
+
+class Login extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      email: '',
+      password: '',
+      error: ''
```

Comment:
Side note: we should decide if we want it to be `error` or `err` in `this.state` as a key.  I think it could be confusing down the road if we are naming them differently throughout the files.